apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -487,13 +487,17 @@ function wpcom_display_global_styles_launch_bar( $bar_controls ) {
 				</div>
 				<div class="launch-bar-global-styles-message">
 					<?php
+					$support_url = function_exists( 'localized_wpcom_url' )
+						? localized_wpcom_url( 'https://wordpress.com/support/using-styles/' )
+						: 'https://wordpress.com/support/using-styles/';
```

Comment:
Needs either localization function from Jetpack, or lint-ignore?

Reply:
On wpcomsh we're calling `localized_wpcom_url` without checking if the function exists. I think it's safe to replace this code with just wrapping the support URL since ETK is only used by Simple and Atomic sites.

Reply:
I still like the idea of being defensive in code that crosses repositories. 😅 

But that's cool, it'll mean that for the 2nd line a lint-ignore would be fine and it'll get translated also for Atomic sites.

Reply:
This logic only gets executed in WPCOM context, Atomic sites don't have Global Styles gated. I theory this code will only get executed on WPCOM. But better safe than sorry XD.

Reply:
Why is this code in ETK then in first place at all, and not just on WP.com?